tests/blackbox/stratis_cli_cert.py

```diff
@@ -26,15 +26,17 @@
 DISKS = []
 
 
-def make_test_pool():
+def make_test_pool(pool_disks):
     """
     Create a test pool that will later get destroyed
+    :param list pool_disks: List of disks with which the pool will be created
     :return: Name of the created pool
     """
     pool_name = p_n()
-    (return_code, _, stderr) = exec_test_command(
-        [STRATIS_CLI, "pool", "create", pool_name, DISKS[0]]
-    )
+    command_param = [STRATIS_CLI, "pool", "create", pool_name]
```

Comment:
You can be more succinct here, lines 36 through 39 can all be compressed into:

```
(return_code, _, stderr) = exec_test_command(STRATIS_CLI, "pool", "create", pool_name] + pool_disks)
```

that '+' is the list concatenation operator, very handy.